src/Prima/Pyxis/Shared/InterceptedEvents.elm

```diff
@@ -0,0 +1,296 @@
+module Prima.Pyxis.Shared.InterceptedEvents exposing
```

Comment:
Qui sei andato troppo oltre credo. In pratica hai riscritto: 
https://package.elm-lang.org/packages/elm/html/latest/Html-Events#on
Mi aspettavo più una serie di utils per il decoding dell'oggetto `event` di JS.

Questo file va rimosso.